src/migrateDashboard.ts

```diff
@@ -61,6 +61,61 @@ function addWidgetErrorToReport(
   };
 }
 
+/*
+    ensure that unexpected elements in the dashboard and page filter objects are removed, this will prevent avoidable migration failures
+    Also keep track of what dashboards and pages have had this treatment
+    It is expected that this treatment will have no negative impact because the unexpected elements would have been ignored by AUI4 anyway
+    but keep track for customer's peace of mind
+  */
+const sanitiseFilters = (
+  filters: any[],
+  errorReport: DashboardErrorReport,
+  {
+    doesReportIncludeStacks,
+    pageKey,
+    pageName,
+  }: {
+    doesReportIncludeStacks?: boolean;
+    pageKey: string;
+    pageName: string;
```

Comment:
Following the same idea, I would make those optional, as it is not needed for dashboard filters.